typescript/examples/vercel-ai/viem/index.ts

```diff
@@ -8,13 +8,16 @@ import { sepolia } from "viem/chains";
 
 import { getOnChainTools } from "@goat-sdk/adapter-vercel-ai";
 import { PEPE, USDC, erc20 } from "@goat-sdk/plugin-erc20";
+import { superfluid } from "@goat-sdk/plugin-superfluid";
 
 import { sendETH } from "@goat-sdk/core";
```

Comment:
I wouldn't change the `viem` example. Can you create a new one for `superfluid`?